resources/language/German/strings.po

```diff
@@ -474,7 +474,7 @@ msgstr "Empfange Wiedergabefortschritt der Episoden von Trakt"
 
 msgctxt "#32120"
 msgid "Parsing episode playback progress %i of %i from Trakt"
-msgstr "Verarbeite Wiedergabefortschritt der Episoden %i of %i von Trakt"
+msgstr "Verarbeite Wiedergabefortschritt der Episoden %i von %i von Trakt"
```

Comment:
Would probably make more sense to have the trakt in the first part of the sentence